src/main/java/InputProcess.kt

```diff
@@ -0,0 +1,278 @@
+import BechTools.BASE64_FORMAT
+import BechTools.BINARY_FORMAT
+import BechTools.DEC_FORMAT
+import BechTools.HEX_FORMAT
+import BechTools.NO_FORMAT
+
+// Written by Mikita 7.4.2022
+/*
+When function returned 'false' – it means FAIL
+'true' – successful result
+ */
```

Comment:
I would suggest to use KDoc for documentation comments (@author Mikita except of Wrriten by..). KDoc is equivalent for JavaDocs in Java.